docs/design/ovs-pipeline.md

```diff
@@ -192,32 +218,37 @@ on the same Node and they have the following IP addresses:
 
 ## Tables
 
+If feature AntreaProxy is enabled, the pipeline is as the following:
+
+![OVS pipeline](../assets/ovs-pipeline-antrea-proxy.svg)
+
+If feature AntreaProxy is disabled, the pipeline is as the following:
+
 ![OVS pipeline](../assets/ovs-pipeline.svg)
```

Comment:
How about adding a separate section under tables that talks about the case when kube-proxy is used, and put the diagram and DNATTable there?
@antoninbas : thoughts?

Reply:
yes, that sounds good to me. We should just emphasize the default case, which is AntreaProxy enabled.